gravitee-apim-console-webui/src/management/settings/top-apis/migrated/top-apis.component.html

```diff
@@ -0,0 +1,122 @@
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<mat-card>
+  <mat-card-content>
+    <div class="header">
+      <div class="header__title">
+        <h3>Top APIs</h3>
+        <p>Showcase your APIs on the Developer Portal homepage</p>
+      </div>
+
+      <div class="header__actions">
+        <button
+          class="add-button"
+          (click)="addTopApi()"
+          mat-raised-button
+          color="primary"
+          *gioPermission="{ anyOf: ['environment-settings-c'] }"
+        >
+          <mat-icon>add</mat-icon>
+          Add Top API
+        </button>
+      </div>
+    </div>
+
+    <table mat-table [dataSource]="topApisList">
+      <!-- Picture Column     -->
+      <ng-container matColumnDef="picture">
+        <th mat-header-cell *matHeaderCellDef id="picture"></th>
+        <td class="picture-cell" mat-cell *matCellDef="let element">
+          <gio-avatar
+            [src]="element.picture_url"
+            [name]="element.name + ' ' + element.version"
+            [size]="32"
+            [roundedBorder]="true"
+          ></gio-avatar>
+        </td>
+      </ng-container>
+
+      <!-- Name Column -->
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef>Name</th>
+        <td mat-cell *matCellDef="let element">{{ element.name }}</td>
+      </ng-container>
+
+      <!-- Version Column -->
+      <ng-container matColumnDef="version">
+        <th mat-header-cell *matHeaderCellDef>Version</th>
+        <td mat-cell *matCellDef="let element">{{ element.version }}</td>
+      </ng-container>
+
+      <!-- Description Column -->
+      <ng-container matColumnDef="description">
+        <th mat-header-cell *matHeaderCellDef>Description</th>
+        <td mat-cell *matCellDef="let element">{{ element.description }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="actions" class="actions">
+        <th mat-header-cell *matHeaderCellDef id="actions"></th>
+        <td mat-cell *matCellDef="let element">
+          <div class="actions__edit">
+            <ng-container *ngIf="topApisList.length > 1">
+              <div class="order-buttons" *gioPermission="{ anyOf: ['environment-top_apis-u'] }">
+                <button
+                  [disabled]="isFirst(element.order)"
+                  (click)="moveUp(element)"
+                  mat-icon-button
+                  aria-label="Button to move up"
```

Comment:
Should be more specific to the action: `Move Top API up`. Same thing with the `aria-label` on the down button and the delete button.